spec/unit/facter/postfix_version_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Facter::Util::Fact.to_s do
+  before(:each) do
+    Facter.clear
+  end
+
+  describe 'postfix_version' do
+    context 'with value' do
+      before :each do
+        Facter::Core::Execution.stubs(:which).with('postconf').returns('/usr/sbin/postconf')
```

Comment:
The tests fail because you're stubbing the wrong class:
```suggestion
        Facter::Util::Resolution.stubs(:which).with('postconf').returns('/usr/sbin/postconf')
```